woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Contents: Fix missing custom background

With this PR using WordPress 5.9 beta, with the Gutenberg disabled, is visible the custom background if it is set. 


Fixes #5416



### Testing
### Manual Testing

Check out this branch

1. Update to WordPress 5.9 beta.
2. Disable Gutenberg.
3. Go to Appearance > Site Editor > Template Parts > Mini Cart.
4. Select the Mini Cart Contents block, change the background.
5. See the background changed in the site editor.
6. Save the block, check on the fronted side that the custom background is visible
